bigquery/google/cloud/bigquery/table.py

```diff
@@ -473,8 +473,9 @@ def _build_resource(self):
             resource['schema'] = {
                 'fields': _build_schema_resource(self._schema)
             }
-        else:
-            raise ValueError("Set either 'view_query' or 'schema'.")
+        elif self.partitioning_type is None:
+            raise ValueError(
+                "Set either 'view_query' or 'schema' or 'partitioning_type'.")
```

Comment:
That makes sense. So if I remove the checks and their tests, is this all you need to approve the change? 

These three lines would be removed from table.py:

```python
        elif self.partitioning_type is None:
            raise ValueError(
                "Set either 'view_query' or 'schema' or 'partitioning_type'.")

```